docs/upgrade.html

```diff
@@ -69,7 +69,9 @@ <h5><a id="upgrade_300_notable" href="#upgrade_300_notable">Notable changes in 3
             <code>org.apache.kafka.clients.consumer.ConsumerPartitionAssignor</code> instead.</li>
         <li>The <code>quota.producer.default</code> and <code>quota.consumer.default</code> configurations were removed (<a href="https://issues.apache.org/jira/browse/KAFKA-12591">KAFKA-12591</a>).
             Dynamic quota defaults must be used instead.</li>
-        <li>The default value for the consumer configuration <code>session.timeout.ms</code> was increased from 10s to 45s. See 
+        <li>The deprecated worker configurations <code>rest.host.name</code> and <code>rest.port</code> were removed (<a href="https://issues.apache.org/jira/browse/KAFKA-12482">KAFKA-12482</a>) in the Kafka Connect.
```

Comment:
> ... in the Kafka Connect.

This seems like a fragment. Perhaps "in the Kafka Connect worker configuration"?

And probably should be "removed ... from" rather than "removed ... in".

Reply:
Also, should probably add `connect` to the list on line 32.